dev/ci/ci-basic-overlay.sh

```diff
@@ -219,7 +213,7 @@ project equations "https://github.com/mattam82/Coq-Equations" "main"
 ########################################################################
 project elpi "https://github.com/LPCIC/coq-elpi" "coq-master"
 
-project hierarchy_builder "https://github.com/math-comp/hierarchy-builder" "coq-master"
+project hierarchy_builder "https://github.com/math-comp/hierarchy-builder" "master"
```

Comment:
Shouldn't hb coq-master be updated instead?

Reply:
The coq-master branch of HB is no longer useful, I take the opportunity of this PR to replace it (c.f. https://github.com/math-comp/hierarchy-builder/pull/381#issuecomment-1697253073 ), sorry I should have written it in the commit message.

Reply:
So HB master is tested with Coq master in its CI and will receive the overlays?

Reply:
HB master will receive the overlay, we will delete the coq-master branch and add Coq master to the CI of the master branch.